src/PlatformCrypto.h

```diff
@@ -3,6 +3,12 @@
 #include <stdbool.h>
 
 #ifdef USE_MBEDTLS
+#include <mbedtls/version.h>
+
+#if MBEDTLS_VERSION_MAJOR >= 3
+#define MBEDTLS_ALLOW_PRIVATE_ACCESS
```

Comment:
I'd rather not depend on private fields. The [3.0 migration guide](https://github.com/Mbed-TLS/mbedtls/blob/development/docs/3.0-migration-guide.md#most-structure-fields-are-now-private) says the struct layout may change in incompatible ways even across minor versions.

Reply:
That's a good point. I reverted to the original commit.